addon/templates/components/feature-controls.hbs

```diff
@@ -25,9 +30,9 @@
 </table>
 
 {{#if showRefresh}}
-  <button {{action 'refresh'}} data-test-button-refresh>Refresh</button>
+  <button {{action "refresh"}} data-test-button-refresh>Refresh</button>
```

Comment:
here I supposed this is normal to get double quotes (eslint with .hbs files)

Reply:
Yes it's the new template linter that force that by default